docs/user-guide/src/version_support.md

```diff
@@ -102,16 +105,16 @@ The table describes which branches/image-tags are tested in each periodic CI tes
 | -------------------------------------------------------------- | ------------ | ----------- | -------------- | -------------- | ----------- | ---------- |
 | metal3-periodic-ubuntu/centos-e2e-integration-test-main        | main         | main        | main           | latest         | latest      | latest     |
 | metal3_periodic_main_integration_test_ubuntu/centos            | main         | main        | main           | latest         | latest      | latest     |
+| metal3-periodic-ubuntu/centos-e2e-integration-test-release-1-8 | release-1.8  | release-1.8 | release-0.7    | v0.7.0         | latest      | v25.0.0    |
 | metal3-periodic-ubuntu/centos-e2e-integration-test-release-1-7 | release-1.7  | release-1.7 | release-0.6    | v0.6.1         | latest      | v24.1.1    |
 | metal3-periodic-ubuntu/centos-e2e-integration-test-release-1-6 | release-1.6  | release-1.6 | release-0.5    | v0.5.0         | latest      | v24.0.0    |
-| metal3-periodic-ubuntu/centos-e2e-integration-test-release-1-5 | release-1.5  | release-1.5 | release-0.5    | v0.5.0         | latest      | v23.1.0    |
 
 | FEATURE AND E2E TESTS                                          | CAPM3 branch | IPAM branch | BMO branch/tag | Keepalived tag | MariaDB tag | Ironic tag |
 | -------------------------------------------------------------- | ------------ | ----------- | -------------- | -------------- | ----------- | ---------- |
 | metal3-periodic-ubuntu/centos-e2e-feature-test-main            | main         | main        | main           | latest         | latest      | latest     |
+| metal3-periodic-ubuntu/centos-e2e-feature-test-release-1-8     | release-1.8  | release-1.8 | release-0.7    | v0.7.0         | latest      | v25.0.0    |
```

Comment:
I missed this to change, test triggers are not valid anymore
```
metal3-periodic-ubuntu/centos-e2e-feature-test-release-1-8-pivoting
metal3-periodic-ubuntu/centos-e2e-feature-test-release-1-8-remediation
metal3-periodic-ubuntu/centos-e2e-feature-test-release-1-8-features
```
same goes for other branch.

Reply:
@smoshiur1237 So the `periodic-` part was removed from the names, right?

Reply:
Periodic part was not removed